src/services/codefixes/inferFromUsage.ts

```diff
@@ -280,10 +326,14 @@ namespace ts.codefix {
                     }
                 }
                 if (!types.length) {
-                    return undefined;
+                    return { declaration: parameter };

```

Comment:
Should `isOptional` be set here?

Reply:
If it's not set, then it's falsy, meaning required. I briefly had `isOptional: boolean`, but requiring its specification in all 4 construction locations seemed needless given that it's usually falsy.